.github/workflows/auto-start-ci.yml

```diff
@@ -0,0 +1,62 @@
+---
+name: Auto Start CI
+
+on:
+  push:
+  schedule:
+    # Runs every five minutes (fastest the scheduler can run). Five minutes is
+    # optimistic, it can take longer to run.
+    - cron: "*/5 * * * *"
+
+jobs:
+  commitQueue:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      # Install dependencies
+      - name: Install jq
+        run: sudo apt-get install jq -y
+      - name: Install Node.js
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: '12'
+      - name: Install node-core-utils
+        run: npm install -g 'https://github.com/mmarchini/node-core-utils#start-ci'
```

Comment:
why install via this link?

Reply:
Still waiting for the PR which adds `ncu-ci start` to land and to be released. It's one of the blockers (unless we want this feature quicker)